docs/guide.md

```diff
@@ -0,0 +1,210 @@
+
+You can put anything you'd like for the documentation page in the Markdown file, but a good starting point is this template:
+
+```md
+---
+title: Buttons
+---
+
+# Buttons
+Insert your documentation here
+
+## Example
+<div class="library__example">
+  Insert your live example here
+</div>
+
+## Code
+
+Insert your code example here
+TODO: INSERT BACKTICKS - HOW DO I ESCAPE THEM??
+```
+
+The `<div class="library__example">` strips out the default PatternPack styling from your example.
+
+The matching Sass/LESS file(s) will be globbed into a file called `_patternpack-patterns.scss or .less`. By default when you generate PatternPack for the first time, you will also get a `patterns.scss or .less` file (or what you configure in [`options.css.fileName`](https://github.com/patternpack/patternpack#cssfilename)) that will `@import` the `_patternpack-patterns` file.
+
+## Releasing Your Code
+PatternPack's most powerful feature is the ability to easily version and release your code, which allows other applications to pull in your styles and lock them to a specific version, preventing unexpected changes.
+
+*Note: The versioning and release process is built on the assumption you are using Git as your source control system. At this time, SVN, Mercurial, TFS (without Git), and others are not supported. However, because the assumption is that PatternPack will live in its own repository, you can set up a separate Git-based project without interrupting your app's development workflow.*
+
+In the default `grunt` configuration, you have a `patternpack:run` task. This task does a few things:
+
+1. Generates a new build in the `./dist` directory (or whatever you configured in [`options.release`](https://github.com/patternpack/patternpack#release)).
+1. Bumps the version number in the `package.json` file
+1. Commits all files in a new commit titled `Release v#.#.#`
+1. Tags the commit `v#.#.#`
+
+The release process follows [Semantic Versioning principles](http://semver.org/).
+
+### Adding All Release Tasks
+By default, the `patternpack:release` task does a patch (i.e., `v1.0.x`) release. To enable minor, and major releases you'll have to modify your `gruntfile.js` to include those tasks:
+
+```
+```js
+module.exports = function (grunt) {
+  grunt.initConfig({
+    patternpack: {
+      run: {},
+      build: {},
+      release: {},
+      "release-patch": {},
+      "release-minor": {},
+      "release-major": {}
+    }
+  });
+
+  grunt.loadNpmTasks('patternpack');
+
+  grunt.registerTask('default', ['patternpack:run']);
+}
+```
+
+The three new tasks, `patternpack:release-patch`, `patternpack:release-minor`, `patternpack:release-major` can be run to increment the desired version.
+
+### Creating a Release
+Before running the command, you'll want a clean working copy (i.e., no uncommitted changes) since the release tasks will stage and commit all files (it runs the equivalent of `git add . && git commit`).
```

Comment:
You may want to provide a quick list of bash commands that will create the release

```base
$ git status
$ grunt release
$ git push --follow-tags
```